test/conversion.ts

```diff
@@ -229,4 +229,47 @@ describe('Conversion', function() {
     setTimeout(check, 150);
     setTimeout(check, 200);
   });
+
+  it('applies domProps to element when given', done => {
+    function main(sources: {react: ReactSource}) {
+      const inc$ = xs.create({
+        start(listener: any) {
+          setTimeout(() => {
+            sources.react
+              .select('button')
+              .events('press')
+              .addListener(listener);
+          }, 30);
+        },
+        stop() {},
+      });
+      const count$ = inc$.fold((acc: number, x: any) => acc + x.target.blah, 0);
+      const vdom$ = count$.map((i: number) =>
+        h(Touchable, {sel: 'button', domProps: {blah: 3}}, [h('div', [h('h1', {}, '' + i)])]),
+      );
+      return {react: vdom$};
+    }
+
+    let turn = 0;
+    const RootComponent = makeCycleReactComponent(() => {
+      const source = new ReactSource();
+      const sink = main({react: source}).react;
+      return {source, sink};
+    });
+    const r = renderer.create(createElement(RootComponent));
+    const root = r.root;
+    const check = () => {
+      const to = root.findByType(Touchable);
+      const view = to.props.children;
+      const text = view.props.children;
+      assert.strictEqual(text.props.children, `${turn * 3}`);
```

Comment:
Was this assert meant to be `turn * 3` or just `3`? I think `turn` is zero, so this assert might be checking that it's zero, in which case it didn't test for the `x.target.blah` existing.

Reply:
Thanks for pointing that out.  Total bug which gave the illusion of passing tests.  I've removed that test in my latest update.